config/locales/administrate.ar.yml

```diff
@@ -5,8 +5,9 @@ ar:
       confirm: "هل أنت متأكد ؟"
       destroy: "حذف"
       edit: "تعديل"
-      show: "إظهار"
-      new: "جديد"
+      edit_item: "تعديل %{item}"
+      show: "إظهار %{item}"
+      new: "جديد  %{resource}"
```

Comment:
hm.. 🤔 This actually might be the other way around, name/item before the string... (but I don't know, since I personally don't speak arabic could check with a friend who does though).

Reply:
I had a chat with a colleague about this and apparently it is correct. You'll read it right-to-left, but because it should be rendered that way it'll make sense.

I'd be happy to be shown we're wrong though!